crates/rethnet_rpc_server/src/lib.rs

```diff
@@ -73,6 +75,15 @@ fn error_response_data<T>(msg: &str) -> ResponseData<T> {
 pub struct Config {
```

Comment:
Move `Config` and `AccountConfig` to a separate `config.rs` file to keep things modular and avoid getting too large files

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/4149/commits/8b6b87a082ec49c0236a9eb5f22c9821ecb516c1